pages/feed/index.tsx

```diff
@@ -111,7 +124,19 @@ const Feeds: WithPageLayout<HighlightSSRProps> = (props: HighlightSSRProps) => {
         image={props?.ogImage}
         twitterCard="summary_large_image"
       />
-      <div className="container flex flex-col gap-12 px-2 pt-12 mx-auto md:px-16 lg:justify-end md:flex-row">
+      <div className="container flex flex-col gap-16 px-2 pt-12 mx-auto md:px-16 lg:justify-end md:flex-row">
+        <div className="flex-col flex-1 hidden gap-8 mt-12 md:flex">
```

Comment:
Why do we need this empty wrapper, it will be renderer in the DOM, `hidden`  I think means `display:none`  can we check on the top of this wrapper, and at line `127` to pass dynamically if the user is logged??

Reply:
That container is a wrapper for feature components... it's would house components that don't need auth to display.

Reply:
Ok Then